arch/mips/src/pic32mx/pic32mx-usbdev.c

```diff
@@ -2347,25 +2405,25 @@ static void pic32mx_ep0setup(struct pic32mx_usbdev_s *priv)
   /* At this point, the request has been handled and there are three possible
    * outcomes:
    *
-   * 1. The setup request was successfully handled above and a response packet
-   *    must be sent (may be a zero length packet).
+   * 1. The setup request was successfully handled above and a response
+   *    packet must be sent (may be a zero length packet).
    * 2. The request was successfully handled by the class implementation.  In
-   *    case, the EP0 IN response has already been queued and the local variable
-   *    'dispatched' will be set to true and ctrlstate != CTRLSTATE_STALL;
-   * 3. An error was detected in either the above logic or by the class implementation
-   *    logic.  In either case, priv->state will be set CTRLSTATE_STALL
-   *    to indicate this case.
+   *    variable case, the EP0 IN response has already been queued and the
+   *    local dispatched will be set to true and ctrlstate != CTRLSTATE_STALL
```

Comment:
I this the word 'this' is missing beween "In case"

The word variable belongs on line 2412 after local

The single quotes around 'dispatched' are really necessary to make the sense readable.  Otherwise, it looks like a word in the sentence, not a variable name

Reply:
Yes, I move variable to the wrong location, I will provide a patch fix it.

Reply:
@yamt and @papatience done.